code/network/multimsgs.cpp

```diff
@@ -3370,6 +3370,8 @@ void send_turret_fired_packet( int ship_objnum, int subsys_index, int weapon_obj
 	else {
 		ADD_FLOAT(ZERO_VALUE);
 	}
+
+	ADD_FLOAT(dist_to_target);
```

Comment:
This is an API change and thus needs to increment the multi version number at some point in this PR.